python_modules/dagster/dagster/core/asset_defs/assets.py

```diff
@@ -52,6 +52,47 @@ def __init__(
     def __call__(self, *args, **kwargs):
         return self._node_def(*args, **kwargs)
 
+    @staticmethod
+    def from_graph(
```

Comment:
might be worth tossing an @experimental on this + a comment describing the parameters/return type